runtime/vm/jnimisc.cpp

```diff
@@ -394,11 +394,11 @@ getObjectClass(JNIEnv *env, jobject obj)
 jint JNICALL
 getVersion(JNIEnv *env)
 {
-	if (J2SE_VERSION(((J9VMThread*)env)->javaVM) >= J2SE_V11) {
+#if JAVA_SPEC_VERSION >= 10
 		return JNI_VERSION_10;
-	} else {
+#else /* JAVA_SPEC_VERSION >= 10 */
 		return JNI_VERSION_1_8;
```

Comment:
Please fix the indentation: these `return` statements are no longer nested within an `if`.